src/ape_console/_cli.py

```diff
@@ -124,11 +125,17 @@ def console(project=None, verbose=None, extra_locals=None):
     if console_extras:
         namespace.update(console_extras)
 
-    from traitlets.config.loader import Config
-
-    config = Config()
-
+    _config = IPythonConfig()
     if environ.get("APE_TESTING"):
-        config.HistoryManager.enabled = False
-
-    IPython.embed(colors="Neutral", banner1=banner, user_ns=namespace, config=config)
+        _config.HistoryManager.enabled = False
+
+    shared_kwargs = {
+        "colors": "Neutral",
+        "banner1": banner,
+        "user_ns": namespace,
+        "config": _config,
+    }
+    if embed:
+        IPython.embed(**shared_kwargs)
```

Comment:
One caveat is that you won't get these on consoles started from `-I` for technical reasons beyond our control.
However, you can manually load them via:

```
%load_ext ape_console.plugin
```

Reply:
I just put this in the docs too